pkg/dashboards/dashboards.go

```diff
@@ -215,6 +215,21 @@ func createUniqId(dname string) string {
 	return newId
 }
 
+// method to check if the dashboard name already exists
+func dashboardNameExists(dname string, orgid uint64) bool {
+	allDashboardIds, err := getAllDashboardIds(orgid)
+	if err != nil {
+		log.Printf("Error getting all dashboard IDs: %v", err)
```

Comment:
nit: use `log.Errorf` here and prepend the function name to the error message, just so it's easier to track down where the log came from if it gets emitted.